tenzir/integration/tests/vast.bats

```diff
@@ -188,14 +188,12 @@ teardown() {
     if [ ! command -v gdate ]; then
       skip "this test requires coreutils to be installed on macOS"
     fi
-    NOW=$(gdate -Ins | tr ',' '.')
+    NOW=$(gdate -Ins)
   else
-    NOW=$(date -Ins | tr ',' '.')
+    NOW=$(date -Ins)
   fi
 
-  # We need subsecond precision here because `date` rounds down otherwise. Also,
-  # we need `tr` because `date` uses a comma for subsecond precision by default
-  # but tenzir requires a dot. (ISO 8601 allows both)
+  # We need subsecond precision here because `date` rounds down otherwise.
```

Comment:
Can you move this comment up to before the first NOW? I think it makes a lot more sense there.